substrate/frame/support/src/traits/tokens/fungible/imbalance.rs

```diff
@@ -141,6 +141,24 @@ impl<B: Balance, OnDrop: HandleImbalanceDrop<B>, OppositeOnDrop: HandleImbalance
 	}
 }
 
+#[cfg(any(feature = "std", feature = "force-debug"))]
+impl<B: Balance, OnDrop: HandleImbalanceDrop<B>, OppositeOnDrop: HandleImbalanceDrop<B>>
+	sp_std::fmt::Debug for Imbalance<B, OnDrop, OppositeOnDrop>
+{
+	fn fmt(&self, fmt: &mut sp_std::fmt::Formatter<'_>) -> sp_std::fmt::Result {
+		fmt.debug_struct("Imbalance").field("amount", &self.amount).finish()
+	}
+}
+
+#[cfg(all(not(feature = "std"), not(feature = "force-debug")))]
```

Comment:
We can probably drop this and just keep the `Debug` implementation all the time

Reply:
we can drop the names of `OnDrop` and `OppositeOnDrop` from output, but they also might be helpful to understand neither this is `Credit` or `Dept`

Reply:
I just meant to not print `wasm:stripped` in wasm.